tests/test_db.py

```diff
@@ -12,7 +12,6 @@
 import responses
 
 from bugbug import db
-from bugbug.db import LastModifiedNotAvailable
```

Comment:
Please don't change this as part of this PR, it makes the PR less readable (ideally each PR should be as self-contained as possible)

Reply:
Ok